pkg/db/tx.go

```diff
@@ -51,16 +49,24 @@ func (d *dbTx) Query(query string, args ...interface{}) (pgx.Rows, error) {
 			"args":  args,
 			"query": queryToString(query),
 		})
+	err = handleSQLError(query, err, log, "query")
```

Comment:
we use this method to convert the error - so call it when you return the error, we already require to do so in order to return.
```go
if err != nil {
	return nil, handleSQLError("query", query, err)
}
```